docs/content/direct/pr-signoff.md

```diff
@@ -1,8 +1,64 @@
+`DCO` file at the root of this Git repository.
+
+In the case of an individual, DCO sign-off is done with a Git "signoff".
+
+We prefer that commits contributed to this repository be signed and
+GitHub verified, but this is not strictly necessary or enforced.
+
+## Commit Sign-off
+
+Your submitted PR must pass the automated checks in order to be reviewed. One of these checks that each commit that you propose to contribute is signed-off. If you use the `git` shell command, this involves passing the `-s` flag on the command line. For other tools, consult their documentation.
+
+### Repairing non-signed-off commits
+
+If you have already created a PR that proposes to merge a branch that
+adds commits that are not signed-off then you can repair this by
+adding the signoff to each using `git commit -s --amend` on each of
+them. Following is an outline of how to do it for a branch that adds
+**exactly one** commit. If your branch adds more than one commit then
+you can extrapolate using `git cherry-pick -s -S` to build up a
+revised series of commits one-by-one.
+
+The following instructions provide a basic walk-through if you have already created your own fork of the repository but yet not made a clone on your workstation. If you have set up your SSH and GPG keys and include `-S` on the `git commit` command line or have configured automatic signing then this will also get the commits signed.
+
+1. Navigate to the **Code** page of the Kubestellar github.
+   
+2. Click the **Fork** dropdown in the top right corner of the page.
+    - Under "Existing Forks" click your fork (should look something like "your_username/kubestellar")
+3. Once in your fork, click the **Code** dropdown.
+    - Under the "Local" tab at the top of the dropdown, select the SSH tab
+    - Copy the SSH repo URL to your clipboard
+4. Open Git Bash (or your CLI of choice), create or change to a different directory if desired.
+5. Clone the repository using `git clone` followed by pasting the URL you just copied.
+6. Change your directory to the Kubestellar repo using `cd kubestellar`.
+7. `git checkout` to the branch in your fork where the changes were committed.
+    - The branch name should be written at the top of your submitted PR page and looks something like "patch-*X*" (where "X" should be the number of PRs made on your fork to date)
+8. Once in your branch, type `git commit -s --amend` to sign off your PR.
+    - You may replace `--amend` with a `-m` followed by a commit message if you desire; the `--amend` simply uses the same commit message as the one you wrote when initially submitting the PR
+    - If prompted with a sign-off page in your Git Bash (or alternative CLI), type `:wq!` to exit the prompt
+9. Type `git push -f origin [branch_name]`, replacing `[branch_name]` with the actual name of your branch.
+10. Navigate back to your PR github page.
+    - A green `dco-signoff: yes` label indicates that your PR is successfully signed
+
```

Comment:
directions in the above section look complete to me based on my experience

Reply:
Thanks.